tests/baselines/reference/completionDetailsOfContextSensitiveParameterNoCrash.baseline

```diff
@@ -1,11 +1,103 @@
+=== /tests/cases/fourslash/completionDetailsOfContextSensitiveParameterNoCrash.ts ===
+// type __ = never;
+// 
+// interface CurriedFunction1<T1, R> {
+//     (): CurriedFunction1<T1, R>;
+//     (t1: T1): R;
+// }
+// interface CurriedFunction2<T1, T2, R> {
+//     (): CurriedFunction2<T1, T2, R>;
+//     (t1: T1): CurriedFunction1<T2, R>;
+//     (t1: __, t2: T2): CurriedFunction1<T1, R>;
+//     (t1: T1, t2: T2): R;
+// }
+// 
+// interface CurriedFunction3<T1, T2, T3, R> {
```

Comment:
What do you think about only including ~10 lines on either side of the marker? Probably not a big deal. Most fourslash tests are brief.

Reply:
There are only a few long tests with 1-2 markers. All the rest are short with 1-2 markers or long with 40+, all next to each other. So it would help only a few tests.